packages/nuxt/src/components/runtime/client-delayed-component.ts

```diff
@@ -0,0 +1,94 @@
+import { createStaticVNode, createVNode, defineComponent, getCurrentInstance, h, onBeforeUnmount, onMounted, ref } from 'vue'
+import type { Component, Ref } from 'vue'
+// import ClientOnly from '#app/components/client-only'
+import { useObserver } from '#app/utils'
+import { getFragmentHTML } from '#app/components/utils'
+import { useNuxtApp } from '#app/nuxt'
+
+// todo find a better way to do it ?

```

Comment:
This is probably the best wait to do it. Can remove this comment :+1:

Reply:
Removed